modules/raveboard/src/app/app.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-root',
+    template: `
+        <div class="root-container">
+            <div class="header">
+                Raveboard &#127940;
```

Comment:
This &#127940 should be a surfer emoji, right? 
Doesn't work on my machine like this. I will replace with the real emoji.

Reply:
Thanks